lab2.js

```diff
@@ -85,28 +112,72 @@ var hello = {
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
+function SentientBeing(planet, language) {
   // TODO: specify a home planet and a language
   // you'll need to add parameters to this constructor
+  this.homePlanet = planet;
+  this.language = language;
 }
 
-// sb is a SentientBeing object
-function sayHello (sb) {
+//sb is a SentientBeing object
+function sayHello(sb) {
     // TODO: say hello prints out (console.log's) hello in the
     // language of the speaker, but returns it in the language
     // of the listener (the sb parameter above).
     // use the 'hello' object at the beginning of this exercise
     // to do the translating
 
+    if (this.homePlanet === 'Earth') {
+      console.log(hello['federation standard']);
+    } else if (this.homePlanet === 'Qu\'onoS') {
+      console.log(hello.klingon);
+    } else if (this.homePlanet === 'Romulus') {
+      console.log(hello.romulan);
+    }
+
+    if (sb.homePlanet === 'Earth') {
```

Comment:
good! This works. And, if you're wanting to make it more concise, think about what you can do with sb.language, this.language, bracket notation, and the hello object.